src/components/Files/FilesTab.tsx

```diff
@@ -651,27 +650,29 @@ export const FilesTab = (props: FilesTabProps) => {
         associatingId={associatingId}
       />
       <Tabs defaultValue={subPage}>
-        <TabsList className="grid w-auto grid-cols-2 w-fit">
-          <TabsTrigger value="all" asChild>
-            <Link
-              className="text-gray-600"
-              href={`/facility/${encounter?.facility.id}/encounter/${encounter?.id}/files/all`}
-            >
-              {t("all")}
-            </Link>
-          </TabsTrigger>
-          <TabsTrigger value="discharge_summary" asChild>
-            <Link
-              className="text-gray-600"
-              href={`/facility/${encounter?.facility.id}/encounter/${encounter?.id}/files/discharge_summary`}
-            >
-              {t("discharge_summary")}
-            </Link>
-          </TabsTrigger>
-        </TabsList>
+        {type === "encounter" && (
+          <TabsList className="grid w-auto grid-cols-2 w-fit">
```

Comment:
aren't the two classes `w-auto` and `w-fit` conflicting? 🤔

Reply:
yep, adjusted 👍